java/connector-node/risingwave-connector-service/pom.xml

```diff
@@ -106,5 +106,15 @@
             <artifactId>risingwave-sink-cassandra</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>com.risingwave</groupId>
+            <artifactId>risingwave-sink-mock-flink-runtime</artifactId>
+            <version>0.1.0-SNAPSHOT</version>
```

Comment:
We can add these dependencies to the dependencyManagement section of the root pom so that we don't have to specify the version number here.